src/__tests__/hooks/useFetchCompareResults.test.tsx

```diff
@@ -1,7 +1,15 @@
+import { Provider } from 'react-redux';
+
 import useFetchCompareResults from '../../hooks/useFetchCompareResults';
-import { StoreProvider } from '../utils/setupTests';
+import { store } from '../utils/setupTests';
 import { renderHook, FetchMockSandbox } from '../utils/test-utils';
 
+function renderFetchCompareResultsHook() {
+  return renderHook(() => useFetchCompareResults(), {
+    wrapper: ({ children }) => <Provider store={store}>{children}</Provider>,
+  });
+}
+
```

Comment:
I decided to remove `StoreProvider` to replace it with this simple function instead. Tell me what you think.